unity-client/Assets/Scripts/MainScripts/DCL/Controllers/MapRenderer/GlobalUsersPositionMarker/MapGlobalUsersPositionMarkerController.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class MapGlobalUsersPositionMarkerController : IDisposable
+{
+    private const float UPDATE_INTERVAL_INITIAL = 10f;
+    private const float UPDATE_INTERVAL_FOREGROUND = 60f;
+    private const float UPDATE_INTERVAL_BACKGROUND = 5 * 60f;
+
+    private const int MAX_MARKERS = 100;
+
+    private const int COMMS_RADIUS_THRESHOLD = 2;
+
+    public enum UpdateMode { FOREGROUND, BACKGROUND }
+
+    FetchScenesHandler fetchScenesHandler;
+    MarkersHandler markersHandler;
+    UserPositionHandler userPositionHandler;
+    Transform markersContainer;
+
+    int commsRadius = 4;
+
+    /// <summary>
+    /// Constructor
+    /// </summary>
+    /// <param name="markerPrefab">prefab for markers</param>
+    /// <param name="overlayContainer">parent for markers</param>
+    /// <param name="coordToMapPosFunc">function to transform coords to map position</param>
+    public MapGlobalUsersPositionMarkerController(UserMarkerObject markerPrefab, Transform overlayContainer, Func<float, float, Vector3> coordToMapPosFunc)
```

Comment:
Why we need to pass a func to convert the coords here? We can't just call the Utils method?

Reply:
asmdef... :'(